src/utils/vcs.rs

```diff
@@ -198,6 +200,11 @@ fn is_matching_url(a: &str, b: &str) -> bool {
     VcsUrl::parse(a) == VcsUrl::parse(b)
 }
 
+pub fn parse_git_url(repo: &str) -> String {
```

Comment:
This function name is not too precise. It does parsing, but it returns only `id`, nothing else.
So maybe a better name would be something like `get_id_from_remote` or something of this sort.

Reply:
@kamilogorek I agree, but I like `get_repo_from_remote` because the `id` in VcsUrl struct returns in the format `"owner-name/repo-name"` which is the repo path.